addons/sourcemod/scripting/include/shavit.inc

```diff
@@ -1631,11 +1631,18 @@ native int Shavit_ReloadReplays(bool restart);
  * Gets time from replay frame that is closest to client.
  *
  * @param 				Client index.
- * @param				Client style.
- * @param				Client track.
  * @return				Replay time.
  */
-native float Shavit_GetClosestReplayTime(int client, int style, int track);
+native float Shavit_GetClosestReplayTime(int client);
```

Comment:
It'd be useful for plugins to be able to check against another styles time without recreating it. e.g. A normal time checking their difference against the segmented time.